+incf/+livescripts/setup.m

```diff
@@ -0,0 +1,72 @@
+%% startup.m
+% Setup variables and automatically download data set from DANDI depending how and "where" the code is ran.
+%
+%% Inputs:
+%
+%    from_location  : the location from where the live-script is ran. Could be either "local", "dandi" or "online".
+%    dandiset_id    : the ID of the DANDI data set.
+%    nwb_path       : the path towards the NWB files within the DANDI data set (keyword argument).
+%    util_libs      : the list of paths relative to the live-script that uses this function that points towards any utils libraries.
+%
+%% Outputs:
+%
+%    dataPath       : the path where the NWB data will be accessible
+%
+%% Example:
+%
+%    nwb_final_path = setup("dandi", "000011", nwb_path="sub-255201_ses-20141124_behavior+ecephys+ogen.nwb", util_libs=["helper-functions"])
+%
+%  This call will setup the live-script to use the 000011 data set, considering that the live-script that requires this setup is running on DANDI-Hub.
+%  This call also setup the NWB file to be located in "000011/sub-255201_ses-20141124_behavior+ecephys+ogen.nwb"
```

Comment:
```suggestion
%    nwb_final_path = setup("dandi", "000011", nwb_path="sub-255201/sub-255201_ses-20141124_behavior+ecephys+ogen.nwb", util_libs=["helper-functions"])
%
%  This call will setup the live-script to use the 000011 data set, considering that the live-script that requires this setup is running on DANDI-Hub.
%  This call also setup the NWB file to be located in "000011/sub-255201/sub-255201_ses-20141124_behavior+ecephys+ogen.nwb"
```

This is a more realistic example using the directory structure returned from `dandi download`